src/platforms/salesforce/client.ts

```diff
@@ -422,6 +455,10 @@ export const client = {
       objectType: 'Lead',
       schema: salesforceLead,
     }),
+    findByEmail: query.findByEmail<typeof salesforceLead>({
```

Comment:
```suggestion
    findByEmail: query.search<typeof salesforceLead>({
```

and then we can manually put `where : { Email : input.email } ` in the unified action